packages/blocks/src/generators/python/actuators.ts

```diff
@@ -101,6 +102,79 @@ function getCodeGenerators(python: MicroPythonGenerator) {
 
 		return `SMALL_OLED_${i2c_channel}.show()\n`;
 	};
+
+	function getSetMotorSpeedName(generator: MicroPythonGenerator): string {
+		const SetMotorSpeedName = generator.provideFunction_("set_motor_speed", [
+			`def ${generator.FUNCTION_NAME_PLACEHOLDER_}(motor, speed):`,
+			"  if speed >= 0:",
+			"    motor.forward(speed)",
+			"  else:",
+			"    motor.backward(-speed)",
+		]);
+
+		return SetMotorSpeedName;
+	}
+
+	function getDCMotorName(
+		generator: MicroPythonGenerator,
+		right: boolean,
+	): string {
+		let name = right ? "right" : "left";
+		let direction_pin = right ? "D2" : "D4";
+		let pwm_pin = right ? "D3" : "D11";
+
+		generator.addImport("leaphymicropython.actuators.dcmotor", "DCMotor");
+
+		const MotorVariableName = python.getVariableName(`dc_motor_${name}`);
+
+		generator.addDefinition(
+			MotorVariableName,
+			`${MotorVariableName} = DCMotor(direction_pin=\"${direction_pin}\", pwn_pin=\"${pwm_pin}\")`,
```

Comment:
`pwn_pin` is a typo, but the same typo exists in the micropython lib, would be good to fix both.